Pods/DBDebugToolkit/DBDebugToolkit/Resources/DBMenuTableViewController.storyboard

```diff
@@ -1,9 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="15702" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" colorMatched="YES" initialViewController="YH8-ay-XNT">
-    <device id="retina4_7" orientation="portrait" appearance="light"/>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="12118" systemVersion="16E195" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" colorMatched="YES" initialViewController="YH8-ay-XNT">
```

Comment:
What has the DBDebugToolkit todo with it?
I think you can reset these changes related to DBDebugToolkit?!

Reply:
I think there is no easy way for selective updates on specific pods other. Only solution I know out of the box would to add a specific version number on each pod we're using. But this way we won't be able to update all pods at once which I don't mind at all.